### PR TITLE
[FSDE-28315] Remove unused variables in Chi vue components

### PR DESCRIPTION
**Pull request by tjimenezLumen**
https://ctl.atlassian.net/browse/FSDE-28315

### CODE REVIEW

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]
In this pull request we must also delete all related `portal` variables, as we do not use them any more.


image

File: src/custom-elements/src/components/tabs/tabs.tsx [hunk 1]
```diff
@@ -377,6 +377,7 @@ export class Tabs {
               data-index={index}
               id={tab.id}
               onMouseEnter={() => this.handlerTabMouseenter(tab)}
+              
             >
```
Comment (josecarlosgonzalezv) [inline src/custom-elements/src/components/tabs/tabs.tsx:381]:
```suggestion
            >
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]
Drawer.tsx still have `@Prop() portal!: boolean;`

File: src/custom-elements/src/components/tabs/tabs.tsx [hunk 2]
```diff
@@ -456,7 +456,6 @@ export class Tabs {
             href="#"
             onClick={e => {
               this.handlerClickTab(e, tab, this.seeMoreTriggerAnchorElement);
-              this.seeMoreDropdown.hide();
```
Comment (josecarlosgonzalezv) [inline src/custom-elements/src/components/tabs/tabs.tsx:459]: Do not remove this line.

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**